server/app.js

```diff
@@ -4,6 +4,7 @@ import bodyPaser from 'body-parser';
 import cors from 'cors';
 import swaggerUI from 'swagger-ui-express';
 import router from './routes/routes';
+import Db from './db/db';
 
 const swaggerDoc = require('../swagger.json');
```

Comment:
ES6 Imports should be observed

Reply:
Okay

Reply:
Rename db.js to index.js, then  change the import to `import Db from './db`